core/src/jsMain/kotlin/Durability.kt

```diff
@@ -0,0 +1,10 @@
+package com.juul.indexeddb
+
+/** https://developer.mozilla.org/en-US/docs/Web/API/IDBTransaction/durability */
+public enum class Durability(
+    internal val jsValue: String
+) {
+    Default("default"),
+    Strict("strict"),
+    Relaxed("relaxed");
+}
```

Comment:
nit: It feels slightly cleaner to me to have an `internal` extension function to provide the `jsValue` string representation. 🤷